examples/video_understanding/configs/workers/task_exit_monitor.yml

```
@@ -0,0 +1 @@
+name: TaskExitMonitor
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)